auto_backup/tests/test_auto_backup.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# © 2015 Agile Business Group <http://www.agilebg.com>
+# © 2015 Alessio Gerace <alesiso.gerace@agilebg.com>
+# © 2016 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import os
+from datetime import datetime
+from openerp.tests import common
+from openerp import exceptions, tools
+
+
+class TestsAutoBackup(common.TransactionCase):
+
+    def setUp(self):
+        super(TestsAutoBackup, self).setUp()
+
+    def new_record(self, method='sftp'):
+        vals = {
+            'name': u'Têst backup',
```

Comment:
Not sure, but I left it assuming that it was testing for handling of special chars in the file name